charts/payment-integration/values.yaml

```diff
@@ -0,0 +1,36 @@
+image:
+  repository: 'MUST_BE_OVERRIDDEN'  # either 'sphereio/commercetools-payone-integration' or 'commercetoolsps/commercetools-paypal-plus-integration/'
+  tag: 'MUST_BE_OVERRIDDEN'  # set tag here
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+service:
+  type: NodePort
```

Comment:
should be `ClusterIP` in the default `values.yaml` to be able to run this chart also easily locally in minicube (override this value in a specific values.yaml if required)

Reply:
d31145a